apps/librelingo_types/librelingo_types/data_types.py

```diff
@@ -1,7 +1,63 @@
 from collections import namedtuple
 
+class TextToSpeechSettings(
+    namedtuple(
+        "TextToSpeechSettings",
+        [
+            "provider",
+            "voice",
+            "engine",
+        ],
+        defaults=["Polly", "Lupe", "standard"]
+    )
+):
+    """
+    Settings about how to use TTS to generate audios
+
+    ### Usage example:
+
+    ```python
+    TextToSpeechSettings(
+        provider="Polly",
+        voice="Aditi",
+        engine="standard"
+    )
+    ```
+    """
+
+    pass
+
+default_text_to_speech_settings = TextToSpeechSettings()
+
+class AudioSettings(
+    namedtuple(
+        "AudioSettings",
+        [
+            "enabled",
+            "text_to_speech_settings",
+        ],
+        defaults=[True, [default_text_to_speech_settings]]
```

Comment:
```suggestion
        defaults=[False, []]
```

what do you think? Wouldn't it be better to have audio disabled by default? That way course authors would need to be more mindful in choosing how they set it up

For text to speech it will need manual setup anyway, no? 🤔